README.md

```diff
@@ -71,6 +70,8 @@ Supported document file extensions: `pdf, doc, docx, ppt, pptx`
 
 Supported image file extensions: `ai, bmp, dcm, eps, gif, png, ps, psd, svs, tga, tif, tiff`
 
+Supported excel extensions: `xls, xlsm, xlsx`
```

Comment:
add to doc types?

Reply:
👍 they fall under the same viewer so that seems better